controls/datagrid/columns/overview.md

```diff
@@ -91,10 +63,50 @@ xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
 
 <snippet id='datagrid-club-model' />
 
+**6.** Add the `BoolToValueConverter` to the page's resources:
+
+<snippet id='datagrid-columns-converter' />
+
+# Columns Features
+
+Find below a quick overview of the DataGrid's Columns features.
+
+## Column Headers
+
+The top cell of a column is called Header. Its purpose is to set a caption for the column, which describes the data displayed in it. The .NET MAUI DataGrid provides fully customizable column headers, check [Column Headers]({%slug datagrid-column-header%}) for detailed information on them.
```

Comment:
...check [Column Headers]({%slug datagrid-column-header%}) for detailed information.